client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -736,7 +736,7 @@
                                 <button type="submit"
                                         class="btn btn-primary"
                                         data-ng-if="ServiceContactCtrl.isEditing"
-                                        data-ng-disabled="!ServiceContactCtrl.hasChanges() || ServiceContactCtrl.isUpdating || contactForm.$invalid">
+                                        data-ng-disabled="!ServiceContactCtrl.isEditing || ServiceContactCtrl.isUpdating || contactForm.$invalid">
```

Comment:
Then the controller's method could be removed right?

Reply:
Done :)